Tests/Functional/SingleFailingTestCase/main.swift

```diff
@@ -0,0 +1,28 @@
+// RUN: %swiftc %s -o %built_tests_dir/SingleFailingTestCase
+// RUN: %built_tests_dir/SingleFailingTestCase > %test_output || true
+// RUN: %xctest_checker %test_output %s
+// CHECK: Test Case 'SingleFailingTestCase.test_fails' started.
+// CHECK: .*/Tests/Functional/SingleFailingTestCase/main.swift:24: error: SingleFailingTestCase.test_fails : 
+// CHECK: Test Case 'SingleFailingTestCase.test_fails' failed \(\d+\.\d+ seconds\).
+// CHECK: Executed 1 test, with 1 failure 0 unexpected\) in \d+\.\d+ \(\d+\.\d+\) seconds
+// CHECK: Total executed 1 test, with 1 failure \(0 unexpected\) in \d+\.\d+ \(\d+\.\d+\) seconds
```

Comment:
`FileCheck` uses a special syntax for regular expressions. `xctest_checker` parses everything as a regular expression. As a result, developers need to take extra care to escape certain characters when writing their checks--for example, `1 failure \(0 unexpected\)`.

This is suboptimal, and could be improved in the future, but `xctest_checker` remains a simple program as a result of this decision.